### PR TITLE
[RXDP-64] - View mode local

### PR DESCRIPTION
**Pull request by ilteoood**
#### Checklist

- [X] your branch will not cause merge conflict with `master`
- [X] run `npm test`
- [X] tests are included
- [] the documentation is updated or integrated accordingly with your changes
- [X] the code will follow the lint rules and style of the project
- [X] you are not committing extraneous files or sensible data

### CODE REVIEW

File: webpack.common.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ module.exports = {
     rules: [
       {
         test: /\.js(x?)$/,
-        exclude: /node_modules\/(?!(@mia-platform\/(syntax-highlighter|json-editor)|swagger2openapi|fetch-har))/,
+        exclude: /node_modules\/(?!(@mia-platform\/(syntax-highlighter|json-editor)|swagger2openapi|fetch-har|openapi-sampler))/,
```
[the remark this answers is not in the document]
Comment (ilteoood) [reply webpack.common.js:21]: It is an internal dependency of ReDoc (https://github.com/Redocly/openapi-sampler), which contain a module with a spread operator that must be transpiled.

[the remark this answers is not in the document]
Comment (fredmaggiowski) [reply webpack.common.js:21]: ok

File: packages/api-explorer/src/index.jsx [hunk 2]
```diff
@@ -5,7 +5,6 @@ import React from 'react'
 import { Collapse, Tag, Divider } from 'antd';
 import get from 'lodash.get'
 import extensions from '@mia-platform/oas-extensions'
-import { Helmet } from 'react-helmet'
```
[the remark this answers is not in the document]
Comment (fredmaggiowski) [reply packages/api-explorer/src/index.jsx:8]: it has already been removed

[the remark this answers is not in the document]
Comment (ilteoood) [reply packages/api-explorer/src/index.jsx:8]: I alerady did it: from both package.json and package-lock

**Pull request merged**